hermes/validator/question_generator.py

```
@@ -63,9 +69,10 @@ async def generate_question(
             round_id: int = 0,
             weight_a: int = 70,
             weight_b: int = 30,
-        ) -> tuple[str, dict | None, str | None]:
+            project_frequency: dict = {}
```

Ruff (0.15.7)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)